tests/posix/sem/src/sem.c

```diff
@@ -0,0 +1,33 @@
+/*
```

Comment:
make this a proper test using ztest please.

Reply:
Sure Anas.